libcontainer/rootfs_linux.go

```diff
@@ -45,15 +45,15 @@ func prepareRootfs(pipe io.ReadWriter, config *configs.Config) (err error) {
 		return newSystemErrorWithCause(err, "preparing rootfs")
 	}
 
+	hasCgroupns := config.Namespaces.Contains(configs.NEWCGROUP)
```

Comment:
it's hard to tell from the diff but is this working correctly or is it just a bunch of cgroup mounts when this bool is true?

Reply:
I'm afraid I don't get the question :-\ most probably because I don't know what "working correctly" means in this case :-\

Reply:
if I'm reading the code correctly, when the spec has NEWCGROUP set, this bool will be true, thus this code in `mountToRootfs` will just mount a bunch of cgroup mounts because of this change:

```git
for _, b := range binds {
			if err := mountToRootfs(b, rootfs, mountLabel); err != nil {
				return err
			if enableCgroupns {
```

Reply:
Yes, looks like it, and this is the same thing upstream runc does: https://github.com/opencontainers/runc/blob/master/libcontainer/rootfs_linux.go#L326 (added by https://github.com/opencontainers/runc/commit/df3fa115f97407713ec6ebe37a5ccf13b81dafd2 which is backported here)

Reply:
ok, i see.  Had to read the whole file, the diff makes it hard to see.

Reply:
If you're seeing the diff on github.com, you can click on "Expand up" or "Expand down" arrows to see more context. Here:
![expand](https://user-images.githubusercontent.com/4522509/52739295-6d27ee00-2f85-11e9-8325-570be5ab39df.png)